pkg/template/crud/create.go

```diff
@@ -26,6 +29,10 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
+	// Adds local flags
+	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", f.Localizer.LocalizeByID("crud.cmd.flag.output.description"))
```

Comment:
how is this replaced?

Reply:
Sorry, didn't get you. Can you please elaborate? :)

Reply:
crud.cmd.flag how is that replaced to commandName.cmd.flag etc.

Reply:
No, we're not replacing that with the command name.

Through PR https://github.com/aerogear/charmil/pull/172, I have included a locale file as a template ([Link to the locale template file](https://github.com/aerogear/charmil/blob/main/pkg/template/crud/crud.en.yaml)).

This locale file even contains entries for every flag that is present in the CRUD commands (including [crud.cmd.flag.output.description](https://github.com/aerogear/charmil/blob/90e7e3fd8adbac493a1f3368cebee1bda1264f5e/pkg/template/crud/crud.en.yaml#L73)).

Whenever the `charmil crud` command is run, those 5 CRUD files will be generated in the path specified by user whereas this locale file will be generated in the `locale/en` directory of the CLI.

Reply:
That needs to be fixed